src/diffusion_for_multi_scale_molecular_dynamics/loss/atom_type_loss_calculator.py

```diff
@@ -190,14 +153,15 @@ def get_p_atm1_given_at(
         Returns:
             one-step transition normalized probabilities of dimension [batch_size, num_classes, num_classes]
         """
-        predicted_p_a0_given_at = torch.nn.functional.softmax(predicted_logits, dim=-1)
-        p_atm1_at = cls._get_probability_atm1_given_at_and_a0_like(predicted_p_a0_given_at,
-                                                                   one_hot_at,
-                                                                   q_matrices,
-                                                                   q_bar_matrices,
-                                                                   q_bar_tm1_matrices,
-                                                                   small_epsilon)
-
+        p_atm1_at = get_probability_at_previous_time_step(
+            probability_at_zeroth_timestep=predicted_logits,
+            one_hot_probability_at_current_timestep=one_hot_at,
+            q_matrices=q_matrices,
+            q_bar_matrices=q_bar_matrices,
+            q_bar_tm1_matrices=q_bar_tm1_matrices,
+            small_epsilon=small_epsilon,
+            probability_at_zeroth_timestep_are_onehot=False,
```

Comment:
I think it would be more clear if this was ` probability_at_zeroth_timestep_are_logits=True`

Reply:
Done.